blocks/library/separator/index.js

```diff
@@ -27,8 +29,17 @@ registerBlockType( 'core/separator', {
 		],
 	},
 
-	edit( { className } ) {
-		return <hr className={ className } />;
+	edit( { className, focus } ) {
+		return [
+			focus && (
+				<InspectorControls key="inspector">
+					<BlockDescription>
+						<p>{ __( 'The horizontal rule represents a paragraph-level thematic break, e.g. a scene change in a story, or a transition to another topic within an article.' ) }</p>
```

Comment:
I'm fine with this PR, but just a small note, the user might not know what's the "horizontal rule" since the block's name is "separator".